docs/extensions/extension_manifest.md

```diff
@@ -35,14 +35,23 @@ format. Manifest files consist of the following sections:
 The extension name is used as the subcommand for executing the extension and
 acts as an identifier when referring to it.
 
-The name must use only lowercase alphanumeric characters, hyphens (`-`), or
-underscores (`_`).
+The name is required and must use only lowercase alphanumeric characters,
+hyphens (`-`), or underscores (`_`).
+
+```toml
+name = "hello-world_1"
+```
 
 ## Description
 
 The description is an optional short blurb about the extension. This should be
 plain text (not Markdown).
 
+```toml
+name = "hello-world_1"
```

Comment:
I don't think it's necessary to always provide the name? The goal should be to to have a subsection of the config documented, not to provide a fully working example.